$NetBSD: patch-CodeLite_CMakeLists.txt,v 1.2 2017/02/14 21:31:43 joerg Exp $

Add switch for NetBSD to not fall under "other OS".

--- CodeLite/CMakeLists.txt.orig	2019-05-21 17:52:21.000000000 +0300
+++ CodeLite/CMakeLists.txt
@@ -69,6 +69,8 @@ set( ADDITIONAL_LIBRARIES "" )
 if (UNIX)
     if ( IS_FREEBSD )
         set(ADDITIONAL_LIBRARIES "-lkvm")
+    elseif ( IS_NETBSD )
+        set(ADDITIONAL_LIBRARIES "-lutil")
     elseif ( UNIX AND NOT APPLE )
         set(ADDITIONAL_LIBRARIES "-ldl -lutil")
     else ( )
